cli/printer/text.go

```diff
@@ -188,6 +188,28 @@ func (t *TextPrinter) PrintLoadBalancerHealthCheck(loadBalancer *models.LoadBala
 	return nil
 }
 
+func (t *TextPrinter) PrintLoadBalancerIdleTimeout(loadBalancer *models.LoadBalancer) error {
```

Comment:
this is overkill imo; other update functions just use the default entity printing or just print some sort of a message.

Reply:
This is not entirely true because (here)[https://github.com/quintilesims/layer0/blob/559-jparsons-lb-timeout/cli/command/load_balancer.go#L386] we see an example where the `l0 loadbalancer healthcheck` subcommand, one that updates the health check of a load balancer, has its own text printer and calls it when the update is done.

Reply:
fair point, but I'd also argue that`l0 loadbalancer healthcheck` is poorly designed. 
Regardless, consider my comments on this a [nit]